plugins/gnostic-go-generator/render_client.go

```diff
@@ -15,20 +15,22 @@
 package main
 
 import (
+	"sort"
 	"strings"
 
 	surface "github.com/googleapis/gnostic/surface"
 )
 
 // ParameterList returns a string representation of a method's parameters
 func ParameterList(parametersType *surface.Type) string {
-	result := ""
+	parameters := make([]string, len(parametersType.Fields))
```

Comment:
Please move the initialization of `parameters` inside the `nil` check of `parametersType`. There is an opportunity for a `nil pointer dereference`. It's failing CI as a result.

Thanks!

Reply:
Ah but then the resulting sort and join wouldn't have a reference...

Let's check early instead with 
```go
if parametersType == nil {
  return ""
}
```